package.json

```diff
@@ -102,6 +102,7 @@
     "lodash.find": "^4.6.0",
     "lodash.includes": "^4.3.0",
     "node-fetch": "^1.6.3",
-    "parse-diff": "^0.4.0"
+    "parse-diff": "^0.4.0",
+    "voca": "^1.2.0"
```

Comment:
I did similar before :) there's obvious some cases need to be considered such as size of dep (especially on browser target), nested dependency (if there is huge dependency sit behind, even dep itself is small it can bloat up dependency tree) but this specific lib's fine for both cases - by danger targets node only, as well as voca does not have other dependency. 

I'll leave this up to your call.